app.py

```diff
@@ -1,209 +1,217 @@
-    @apiSuccess (200) {Object} game returns a game
-    @apiSuccessExample {json} Example Response:
-       {
-         "_id": "580bd3896df3906e619cd9f5",
-         "week": 1,
-         "teams": [
-           "Karma Down Under": ["Alison Ward"],
-           "Katie Parity": ["Dan Thomson"]
-         ],
-         "events": [
-           "Pull,Al Colantonio"
-         ],
-         "stats": {
-           "Al Colantonio": {"Pulls": 1, "SalaryDelta": 2000, "Salary": 50000}
-         }
-       }
-    """
-    game = Game.query.get(id)
-    return jsonify(game.to_dict())
-
-# Client
-@app.route('/', defaults={'path': ''})
-@app.route('/<path:path>')
-def client(path):
-    if(path == ""):
-        return send_from_directory(client_path, 'index.html')
+def create_app():
+    # App
+    app = Flask(__name__, static_folder=client_path)
+    if os.name == 'nt':
+        app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///' + db_path
     else:
-        if(os.path.exists(client_path + '/' + path)):
-            return send_from_directory(client_path, path)
-        else:
+        app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:////' + db_path
+
+    # Database
+    db.init_app(app)
+    if (os.path.exists(db_path) == False):
+        with app.app_context():
+            db.create_all()
+
+    # Upload
+    @app.route('/upload', methods=['POST'])
+    def upload():
+        """
+        @api {post} /upload From Stats Keeper Client
+        @apiGroup Upload
+        @apiDescription This is only path which creates new data on the server.
+        The Stats Keeper Client uploads the stats to this endpoint and the server
+        does any required post-processing before saving the game to the database.
+         *
+        @apiParam {String} league the name of the league
+        @apiParam {Number} week the week number of the game
+        @apiParam {Object} teams a key for each team containing an array of player names
+        @apiParam {Array} event_string the array of events recored by the client
+         *
+        @apiParamExample {json} Example Upload:
+            {
+              "league": "ocua_16",
+              "week": 1,
+              "teams": {
+                "Karma Down Under": ["Alison Ward"],
+                "Kindha's Ongoing Disappointments": ["Jen Cluthe"]
+              },
+              "points": [
+                {"type": "PULL", "firstActor": "Alison Ward"}
+              ]
+            }
+        """
+        game = Game()
+        points = json.loads(request.json['points'])['points']
+        game.league = request.json['league']
+        game.week = request.json['week']
+        game.teams = json.dumps(request.json['teams'])
+        game.score = json.dumps(request.json['score'])
+        game.points = json.dumps(points)
+
+        db.session.add(game)
+        db.session.commit()
+
+        stats = StatsCalculator(game.id, points).run()
+        for stat in stats:
+            db.session.add(stat[1])
+        db.session.commit()
```

Comment:
the only real change to this file is right here. The rest is just a whitespace indent so that `create_app` can be exported for testing